src/views/VisitsView.vue

```diff
@@ -71,12 +70,14 @@
             <h2 v-else>
               {{ countMessage }}
             </h2>
-            <p class="search-description" v-html="currentQueryDescription"></p>
             <div
               v-if="!queryPending && !noQueryYet && visits.length > 0"
               class="results"
             >
-              <h1>Visit Summary Per Device</h1>
+              <h1>
+                <span v-if="canLoadMore">(Incomplete)</span> Visit Summary Per
```

Comment:
Maybe we could just write the date here or the number of recordings e.g. Visist Summary for the first 400 recordings or Visit Summary for 2021-02-04 12:00:00 to 2021-03-04 13:00:00